source/_integrations/rotarex.markdown

```diff
@@ -0,0 +1,75 @@
+---
+title: "Rotarex DIMES SRG"
+description: "Instructions on how to integrate your Rotarex gas tanks into Home Assistant."
+documentation: https://rotarexsrg.com/category/digital-measurement-system
+ha_release: 2026.1
+ha_domain: rotarex
+ha_iot_class: cloud_polling
+ha_codeowners:
+  - '@Rotarex'
+---
+
+The **Rotarex DIMES SRG** {% term integration %} allows you to monitor your Rotarex DIMES SRG Camping gas tanks in Home Assistant.
+
+This integration uses the official Rotarex cloud API to fetch the latest data.
+
+## Supported devices
+
+- Rotarex DIMES SRG camping gas tanks added to your Rotarex account
+
+## Prerequisites
+
+1. Install the **Rotarex DIMES SRG Camping** app on your phone.
+2. Create a Rotarex account, then sign in.
+3. Add your tank to the app and confirm it syncs.
+4. Have your app login credentials (email and password) ready.
+
+## Configuration
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+
+[![Open your Home Assistant instance and start setting up a new integration.](https://my.home-assistant.io/badges/config_flow_start.svg)](https://my.home-assistant.io/redirect/config_flow_start/?domain=rotarex)
+
+1. In the pop-up, you will be asked to enter the **email** and **password** you use for the Rotarex DIMES SRG Camping mobile app.
```

Comment:
ah, I meant this format (but with the correct indents):

Email:
    description: "The email address you used to login to your ...."
Password:
    description: "The password you user for..."

<img width="461" height="94" alt="image" src="https://github.com/user-attachments/assets/3e729140-20a8-4145-886c-f9408961c63f" />